lists/global.csv

```diff
@@ -1435,3 +1435,7 @@ https://149.112.112.9:5053/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQA
 https://9.9.9.9:5053/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQAAAQAB,HOST,Hosting and Blogging Platforms,2020-07-16,Chelchela,
 https://doh.dns.sb/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQAAAQAB,HOST,Hosting and Blogging Platforms,2020-07-16,Chelchela,
 https://mozilla.cloudflare-dns.com/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQAAAQAB,HOST,Hosting and Blogging Platforms,2020-07-16,Chelchela,
+https://i2p.net/,ANON,Anonymization and circumvention tools,2020-07-30,Chelchela,
```

Comment:
What is the difference between this site and https://geti2p.net/en/?

Reply:
This is just an outdated edition. And it does not seem to be part of the main site. It seems better to delete it. (?)
![image](https://user-images.githubusercontent.com/12384263/90640938-46f9a300-e239-11ea-8134-fd75eb9ffe34.png)
![image](https://user-images.githubusercontent.com/12384263/90640966-51b43800-e239-11ea-8adf-5d8a3ae746c6.png)